### PR TITLE
Ports the teuthology-pull-requests job to JJB

### PR DESCRIPTION
**Pull request by andrewschoen**
Let me know if we're missing anybody in the white-list or admin-list.  I'm also unsure what the process is to deploy this after merge, so I'll need some guidance there.

### CODE REVIEW

File: teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            - jcsp
+            - gregsfortytwo
+            - GregMeno
+            - dzafman
+            - dillaman
+            - dachary
+            - liewegas
+            - idryomov
+            - vasukulkarni
+          trigger-phrase: ''
```
Comment (andrewschoen) [inline teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml]: Do I need anything here?  Is this where the phrase like "ok to test" would go?

Comment (ktdreyer) [inline teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml]: Yeah, that seems a little odd to leave that blank. Is it blank in the job's `config.xml`?

Comment (alfredodeza) [inline teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml]: I don't think this is needed, it is configured globally in the plugin

Comment (andrewschoen) [inline teuthology-pull-requests/config/definitions/teuthology-pull-requests.yml]: Gotcha, I noticed it was blank in the other *-pull-requests jobs as well.

**Comment by ktdreyer** [comment]
>I'm also unsure what the process is to deploy this after merge, so I'll need some guidance there.

The process is just "merge the PR". Assuming the job has the same name as the previous job, JJB will overwrite the old one.

**Pull request merged**